.circleci/config.yml

```diff
@@ -1133,26 +1137,47 @@ jobs:
         description: The gate to run the acceptance tests against. This gate should be defined in op-acceptance-tests/acceptance-tests.yaml.
         type: string
         default: ""
-      resource_class:
-        description: Machine resource class
-        type: string
-        default: xlarge
       no_output_timeout:
         description: Timeout for when CircleCI kills the job if there's no output
         type: string
         default: 30m
     machine: true
-    resource_class: <<parameters.resource_class>>
+    resource_class: xlarge
     steps:
       - utils/checkout-with-mise
-      - attach_workspace:
-          at: "."
+      - run:
+          name: Setup Kurtosis
+          command: |
```

Comment:
I would just separate this into steps - mise install is step 1, kurtosis engine start is step 2

Reply:
Discussed on Slack; probably redundant in any case due to the earlier mise-install step; removing.